platform/settings.cpp

```diff
@@ -17,375 +17,407 @@
 #include "std/iostream.hpp"
 #include "std/sstream.hpp"
 
-
 #define FIRST_LAUNCH_KEY "FirstLaunchOnDate"
 
-static char const DELIM_CHAR = '=';
+namespace
+{
+char const DELIM_CHAR = '=';
```

Comment:
It’s an old code, is it ok if we reformat it’s constants in a separate request later?
> On 1.3.2016, at 12.02, ygorshenin <notifications@github.com> wrote:
> 
> In platform/settings.cpp <https://github.com/mapsme/omim/pull/2045#discussion_r54538131>:
> 
> >  #define FIRST_LAUNCH_KEY "FirstLaunchOnDate"
> >  
> > -static char const DELIM_CHAR = '=';
> > +namespace
> > +{
> > +char const DELIM_CHAR = '=';
> s/DELIM_CHAR/kDelimChar/g
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2045/files#r54538131>.
>